### ISSUE TITLE
Drop-Based Learning Rate Schedule (LearningRateScheduler)

### ISSUE THREAD

**Issue by javierfs** [matterport/Mask_RCNN#1228]
I want to drop the learning rate at specific times during training. To do so, I implemented a callback in Keras using a LearningRateScheduler. I added the following line in the function train in models.py: 

```
 callbacks = [
            keras.callbacks.TensorBoard(log_dir=self.log_dir,
                                        histogram_freq=0, write_graph=True, write_images=True),
            keras.callbacks.ModelCheckpoint(self.checkpoint_path,
                                            verbose=0, save_weights_only=True),
            # learning schedule callback
            **keras.callbacks.LearningRateScheduler(self.step_decay)**


        ]
```

Furthermore, I defined a function to call within the class MaskRCNN() in models.py that takes the epoch number as an argument and returns the learning rate to use in stochastic gradient descent. 

```
# learning rate schedule
    def step_decay(self, epoch):
        initial_lrate = 0.001
        drop = 0.5
        epochs_drop = 10.0
        lr = initial_lrate * math.pow(drop, math.floor((1 + epoch) / epochs_drop))
        log("\nLR at epoch {} = {}  \n".format(epoch,lr))
        return lr
```

However, when I run the code, it seems to be stuck just after epoch 1 (starting from epoch 0). Anybody knows how to implement this?

**Comment by javierfs** [comment]
I have solved this by implementing the solution from here: https://www.kaggle.com/fergusoci/loss-based-learning-rate-scheduler-for-keras 

However, I do not know how to show the lr decreasing over the epochs.. any ideas?
